python/py-rfc6555/Portfile

```diff
@@ -20,14 +20,21 @@ long_description    Python implementation of the Happy Eyeballs Algorithm descri
 
 homepage            https://github.com/sethmlarson/rfc6555
 
-checksums           rmd160  7bd0464b9353f10892fca2cca5943962ec5f9cfa \
-                    sha256  191cbba0315b53654155321e56a93466f42cd0a474b4f341df4d03264dcb5217 \
-                    size    11218
+checksums           rmd160  5a2e58cbbdbe6134dcd4f0d86867e236081774ab \
+                    sha256  123905b8f68e2bec0c15f321998a262b27e2eaadea29a28bd270021ada411b67 \
+                    size    10094
 
-python.versions     27 38 39
+python.versions     27 35 38 39 310
```

Comment:
I note you've added Python 3.5 support?

Reply:
@pmetzger I missed it originally by mistake. Seems it works with 3.5 since beginning.

Anyway, changelog: https://github.com/sethmlarson/rfc6555/blob/main/CHANGES.rst make it clear.

Reply:
3.5 is no longer supported by the Python devs though. We should be phasing it out.

Reply:
good point